### ISSUE TITLE
Upgrading MySQL tables to utf8mb4 may be required

### ISSUE THREAD

**Issue by marksandford** [Islandora-Collaboration-Group/ISLE-Documentation#93]
https://github.com/Islandora-Collaboration-Group/ISLE/issues/189

**Comment by marksandford** [comment]
Do we need documentation for this right now?  Is this currently handled by a script, or will we need step by step instructions for 1.1?

**Comment by br2490** [comment]
Yes. Need to script this and document for migrators (migrations)
Related to: https://github.com/Islandora-Collaboration-Group/ISLE/issues/189
